homeassistant/components/netgear/strings.json

```diff
@@ -29,5 +29,116 @@
         }
       }
     }
+  },
+  "entity": {
+    "sensor": {
+      "link_type": {
+        "name": "Link type"
+      },
+      "link_rate": {
+        "name": "Link rate"
+      },
+      "signal_strength": {
+        "name": "Signal strength"
+      },
+      "ssid": {
+        "name": "ssid"
+      },
+      "access_point_mac": {
+        "name": "Access point mac"
+      },
+      "upload_today": {
+        "name": "Upload today"
+      },
+      "download_today": {
+        "name": "Download today"
+      },
+      "upload_yesterday": {
+        "name": "Upload yesterday"
+      },
+      "download_yesterday": {
+        "name": "Download yesterday"
+      },
+      "upload_week": {
+        "name": "Upload week"
+      },
+      "upload_week_average": {
+        "name": "Upload week average"
```

Comment:
> Also, is a week the last 7 days or is it literally this week.

Good question, it comes directly from the router, so I will look into this.

Reply:
In the router it is called "This week" and then you have collums for upload/download etc.
I personally do not use these sensors, so I kind of don't know, but since you can set the start date/time when you want it to start measuring I am pretty sure it is the upload since the start day (like monday) and not the past 7 days.

Reply:
Besides I start with Upload/Download such that all the sensors will be shown next to eachother in the device page (sorted alphabetically), to not scatter around the upload/download sensors.

Reply:
I would prefer to keep it like this, mainly for sorting reasons.

Reply:
I added "this" to the week/month upload/download